modules/claims_api/app/models/claims_api/user_info.rb

```diff
@@ -1,19 +1,19 @@
 # frozen_string_literal: true
 
-require 'rest-client'
-
 module ClaimsApi
   class UserInfo
     def initialize(user_info_url, token_string, audience)
       payload = { aud: audience }
-      response = RestClient.post(user_info_url,
-                                 payload,
-                                 { Authorization: "Bearer #{token_string}",
-                                   apiKey: Settings.claims_api.token_validation.api_key })
+      headers = {
+        Authorization: "Bearer #{token_string}",
+        apiKey: Settings.claims_api.token_validation.api_key
+      }
+
+      response = Faraday.post(user_info_url, payload, headers)
```

Comment:
This is probably fine, but generally I'd avoid refactoring code that doesn't belong to us (i.e. anything in `/modules`)